catalog/transformer/catalog-transformer-pptx/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -12,13 +12,18 @@
  *
  **/
  -->
-<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0">
 
     <reference id="tikaInputTransformer" interface="ddf.catalog.transform.InputTransformer" filter="(id=tika)"
                availability="mandatory"/>
 
     <bean id="pptxTransformer" class="ddf.catalog.transformer.input.pptx.PptxInputTransformer">
         <argument ref="tikaInputTransformer"/>
+        <cm:managed-properties
+                persistent-id="ddf.catalog.transformer.input.pptx.PptxInputTransformer"
+                update-strategy="container-managed"/>
+        <argument value="false"/>
```

Comment:
So, by default, you are going to use the metadata for the title? In the pdf input transformer, my default is to use the filename. Should we make these transformers default to the same behavior?